x-pack/platform/plugins/shared/observability_ai_assistant/server/service/index_assets/templates/kb_component_template.ts

```diff
@@ -66,10 +66,6 @@ export function getComponentTemplate(inferenceId: string) {
         'ml.tokens': {
           type: 'rank_features',
         },
-        confidence: keyword,
-        is_correction: {
-          type: 'boolean',
-        },
```

Comment:
here as well, it might be less risky to keep it but add a comment that it is deprecated